### PR TITLE
docs: slightly reword unnamedResult docs

### PR DESCRIPTION
**Pull request by Quasilyte**

### CODE REVIEW

File: docs/overview.md [hunk 1]
```diff
@@ -581,6 +582,8 @@ Gives false-positives for:
 
 
 
+
+
```
Comment (cristaloleg) [inline docs/overview.md]: I think we should fix `makedocs` tool :)

Comment (Quasilyte) [reply docs/overview.md]: Maybe. I don't know why it did that..
Needs investigation and dedicated issue.

Comment (Quasilyte) [reply docs/overview.md]: Could you file an issue?

Comment (cristaloleg) [reply docs/overview.md]: yep

**Pull request merged**